packages/morph/src/morph.js

```diff
@@ -207,7 +207,7 @@ async function patchChildren(from, to) {
             } else {
                 let added = addNodeTo(currentTo, from) || {}
 
-                await breakpoint('Add element: ' + added.outerHTML || added.nodeValue)
+                await breakpoint('Add element: ' + (added.outerHTML || added.nodeValue))
```

Comment:
For text nodes, the nodeValue wasn't outputting, so had to add brackets for it to output properly